app/utilities/dhbDisplayName.ts

```diff
@@ -0,0 +1,36 @@
+type NameMapType = {
+  [key: string]: string;
+};
+
+const nameMap: NameMapType = {
```

Comment:
Can do `const nameMap: Record<string, string>`

Ideally once all this is merged we create a type so we are consistent with the serverside name, so would be something like 

`const nameMap: Record<DhbName, string>`

Reply:
Liiiiit exactly what I'm looking for